wpilibNewCommands/src/test/native/cpp/frc2/command/CommandTestBase.h

```diff
@@ -106,14 +106,18 @@ class CommandTestBaseWithParam : public ::testing::TestWithParam<T> {
  protected:
   CommandScheduler GetScheduler() { return CommandScheduler(); }
 
-  void SetUp() override { frc::sim::DriverStationSim::SetEnabled(true); }
+  void SetUp() override {
+    frc::sim::DriverStationSim::SetEnabled(true);
+    frc::sim::DriverStationSim::NotifyNewData();
```

Comment:
Should it just call SetDSEnabled() here to match Java?

Reply:
Probably, should I also change C++ `CommandTestBase` too?

Reply:
Also, why are the C++ steps split between the ctor and `SetUp()`?

Reply:
The Google test site has some [recommendations on using a constructor vs. `SetUp()`](http://google.github.io/googletest/faq.html#CtorVsSetUp), but it recommends using a constructor except for certain cases that we aren't using.

Reply:
> Also, why are the C++ steps split between the ctor and `SetUp()`?

Because of how google test fixtures are implemented, only one concrete implementation of `SetUp` (and `TearDown`) will be called by google test, but the constructor will *always* be called. This is exactly why this issue occured- the SetUp function from CommandTestBase was overridden by the one from CommandTestBaseWithParam. 

We should either move this code to the constructor of CommandTestBase, or call the overrident setup (from CommandTestBase).

Reply:
`CommandTestBaseWithParam` isn't derived from `CommandTestBase`? (See https://github.com/wpilibsuite/allwpilib/blob/main/wpilibNewCommands/src/test/native/cpp/frc2/command/CommandTestBase.h#L97)

Reply:
yepppp.... my bad.